### PR TITLE
Add e2e test cases for Softmax and LogSoftmax

### PR DESCRIPTION
**Pull request by houseroad**
End to end tests for Softmax and LogSoftmax

### CODE REVIEW

File: test/test_caffe2.py [hunk 1]
```diff
@@ -511,6 +511,19 @@ def forward(self, ma, m1, m2):
         m2 = Variable(torch.randn(4, 5))
         self.run_model_test(MyModel(), train=False, input=(ma, m1, m2), batch_size=BATCH_SIZE, use_gpu=False)
 
+    def test_softmax(self):
+        model = nn.Softmax(dim=2)
+        input = Variable(torch.randn(2, 3, 4).fill_(1),
```
Comment (bddppq) [inline test/test_caffe2.py]: Could you add more cases for ndim?

File: test/test_caffe2.py [hunk 2]
```diff
@@ -511,6 +511,19 @@ def forward(self, ma, m1, m2):
...
+                         requires_grad=True)
+        self.run_model_test(model, train=False, batch_size=BATCH_SIZE, input=input)
+
+    def test_logsoftmax(self):
+        model = nn.LogSoftmax(dim=2)
+        input = Variable(torch.randn(2, 3, 4).fill_(1),
```
Comment (bddppq) [inline test/test_caffe2.py]: ditto
